.tekton/prometheus-pull-request.yaml

```diff
@@ -103,15 +76,15 @@ spec:
         path-context
       name: dockerfile
       type: string
-    - default: "false"
+    - default: "true"
       description: Force rebuild image
       name: rebuild
       type: string
-    - default: "false"
+    - default: "true"
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
+    - default: "true"
       description: Execute the build with network isolation
```

Comment:
Not sure about these changes...

Reply:
@marioferh did you just change all defaults to true? IMHO the only change that we'd need is the one about **build-source-image**. For the other ones, let's just keep the default as-is.